.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,7 @@
+contact_links:
+  - name: Mailing List
+    url:
```

Comment:
@oleg-nenashev  i am sort of confused with the links to put 😅, any help is appreciated